src/pages/settings/Payments/PaymentsPage.js

```diff
@@ -16,15 +18,20 @@ import {PayPal} from '../../../components/Icon/Expensicons';
 import MenuItem from '../../../components/MenuItem';
 import getClickedElementLocation from '../../../libs/getClickedElementLocation';
 import CurrentWalletBalance from '../../../components/CurrentWalletBalance';
+import ONYXKEYS from '../../../ONYXKEYS';
+import Permissions from '../../../libs/Permissions';
 
 const PAYPAL = 'payPalMe';
 
 const propTypes = {
     ...withLocalizePropTypes,
+
+    /** List of betas available to current user */
+    betas: PropTypes.arrayOf(PropTypes.string),
 };
 
 const defaultProps = {
-    payPalMeUsername: '',
```

Comment:
Was this deletion intentional?

Reply:
Yeah afaict it isn't used in that file. Did i miss it somewhere?

Reply:
You're right, it doesn't look like we're passing this component any props whatsoever.